### PR TITLE
Update h5bp

### PR DESCRIPTION
**Pull request by TangRufus**
Note: I can't get [ssl.no-default](https://github.com/h5bp/server-configs-nginx/blob/master/sites-available/ssl.no-default) to work. Thus, I omitted it.

### CODE REVIEW

File: roles/nginx/templates/nginx.conf.j2 [hunk 1]
```diff
@@ -1,57 +1,60 @@
...
+# This is only used when you don't override it on a server{} level
+# Default: logs/error.log error
 error_log  {{ nginx_logs_root }}/error.log warn;
 {% endblock %}
 
 {% block pid %}
 # The file storing the process ID of the main process
-pid        /run/nginx.pid;
+# Default: nginx.pid
+pid        /var/run/nginx.pid;
```
Comment (swalkinshaw) [inline roles/nginx/templates/nginx.conf.j2]: Was this change from h5bp? I believe it should stay as `/run`. It's the "newer" location and `/var/run` is symlinked to it.

Comment (TangRufus) [reply roles/nginx/templates/nginx.conf.j2]: it's from h5bp:
https://github.com/h5bp/server-configs-nginx/blob/c5c6602232e0976d9e69d69874aa84d2a2698265/nginx.conf#L34

Comment (TangRufus) [reply roles/nginx/templates/nginx.conf.j2]: So `/run` is the correct one?

Comment (swalkinshaw) [reply roles/nginx/templates/nginx.conf.j2]: From what I can tell it is. So let's keep it as `/run`

Comment (TangRufus) [reply roles/nginx/templates/nginx.conf.j2]: reverted

**Comment by swalkinshaw** [comment]
Rest looks good, thanks 👍

**Pull request merged**